odoo/addons/base/models/res_company.py

```diff
@@ -213,15 +213,18 @@ def _compute_uninstalled_l10n_module_ids(self):
                AND country.id = ANY(%(country_ids)s)
           GROUP BY country.id
         """, {
-            'country_ids': self.country_id.ids,
+            'country_ids': self.partner_id.country_id.ids,
```

Comment:
Why?

Reply:
`partner_autocomplete` test cases were failing when I tried to debug it whenever your try using `companies.country_id` under `create` method the country_id was not set may be due to it's a compute field better to use `partner_id.country_id` at least it is a stored field you can rely on

Reply:
It is because we are trying filter out `uninstalled_l10n_module_ids`

Reply:
That comment is 8 years old...
Some things might have changed since.
```python
@api.depends(lambda self: ['partner_id.%s' % fname for fname in self._get_company_address_field_names()])
```